backend/globaleaks/handlers/wizard.py

```diff
@@ -62,6 +87,9 @@ def db_wizard(session, tid, hostname, request):
         if  tid != 1 and root_tenant_node.get_val('crypto_escrow_pub_key'):
             node.set_val('crypto_escrow_prv_key', Base64Encoder.encode(GCE.asymmetric_encrypt(root_tenant_node.get_val('crypto_escrow_pub_key'), crypto_escrow_prv_key)))
 
+    admin_user = None
+    is_tenant_anac = False
```

Comment:
this variable is project specific, and will need to be abstracted in relation to a functionalty